src/main/java/org/craftercms/studio/impl/v1/service/site/SiteServiceImpl.java

```diff
@@ -165,6 +161,8 @@ public class SiteServiceImpl implements SiteService, ApplicationContextAware {
 
 	protected UserDAO userDao;
 
+	SqlSessionFactory sqlSessionFactory;
```

Comment:
Should this be private or protected ?

Reply:
Updated to use `protected` to align with this class implementation.